doc/modules/classes.rst

```diff
@@ -655,8 +655,9 @@ Kernels:
    :template: class.rst
 
    impute.SimpleImputer
+   impute.IterativeImputer
    impute.MissingIndicator
-
+   
```

Comment:
Could you remove this

Reply:
@jnothman I assume you don't want me to make a PR for this =)

Reply:
Oh sorry I did not see it was @jnothman PR. I'll fix it then :)